README.md

```diff
@@ -5,9 +5,11 @@ This project is where the [Open Container Initiative](http://www.opencontainers.
 Table of Contents
 
 - [Filesystem Bundle](bundle.md)
-- [Container Configuration](config.md)
+- [Host-independent Container Configuration](config.md)
   - [Linux Specific Configuration](config-linux.md)
-- [Runtime and Lifecycle](runtime.md)
+- [Host-specific Container Configuration](runtime.md)
+  - [Linux Specific Configuration](runtime-linux.md)
```

Comment:
you have two `Linux Specific Configuration` one for the `config-linux.md` and the other for `runtime-linux.md`